app/domain/ca/x509/issuer.rb

```diff
@@ -0,0 +1,49 @@
+module CA
+  # :reek:UncommunicativeModuleName
+  module X509
+    # Issuer represent the CA signing key material stored in Conjur
+    class Issuer < Dry::Struct::Value
+      class << self
+        include ::CA::AnnotationLoader::RsaPrivateKey
+        include ::CA::AnnotationLoader::MaxTTL
```

Comment:
You have just taken the first step down the road to perdition :)

Consider `include` to be an anti-pattern.  If you need to share functionality, do it by composing objects.  Ie, object A and B both need some functionality.  Create the functionality in C, and pass C to both A and B.  Whenever you want to reach for inheritance, reach for this instead.

You probably know that, though.  I'm guessing in this case the issue was that you re-using stuff at the class level.  That doesn't really change anything conceptually though, and we still have better options.  

More to come.